package/gluon-core/luasrc/lib/gluon/upgrade/010-primary-mac

```diff
@@ -76,6 +76,7 @@ local primary_addrs = {
 			'ocedo,raccoon',
 			'tplink,archer-a7-v5',
 			'tplink,archer-c2-v3',
+			'tplink,archer-c5-v1',
```

Comment:
This should not be necessary. The Archer C5 v1 uses the same network setup as the C7 v2, and both define the same interface for their label MAC in the Device Tree.

Reply:
resolved in 4485febc9f87709286e90d7f19fd7f5943dfa20e